src/components/pages/Theme/Theme.css

```diff
@@ -20,3 +20,26 @@
 .dropdownBox .dropdown {
   margin: 5px 0;
 }
+
+i.errorIcon {
+  color: var(--alert-error-900);
```

Comment:
We should be able to pass in a `color` prop to the icons instead of styling them.

Reply:
Do we want the colour to be exact the same as we were using? Or since this is just the demo of the `Table` passing `green` or `red` should be ok?

Reply:
I just saw some other icons where we just pass the `colour` so I did the same with these icons from the table.

Reply:
Nice. I'm going to create a separate ticket for us to style the icon colors using the GA colors. I think that will be a good task for @OLyMPiaNAssIS to start on next week.